package.json

```diff
@@ -82,7 +83,7 @@
     "lodash.set": "4.3.2",
     "lodash.unset": "4.5.2",
     "markdown-to-jsx": "7.3.2",
-    "next": "^14.2.0-canary.38",
```

Comment:
Can bump [14.2 now](https://github.com/vercel/next.js/releases/tag/v14.2.2)

Reply:
The version isn't pinned so it should grab the latest patch version on build.

Reply:
Ya I think it should be pinned.

Reply:
Pinned